vars/annotateTag.groovy

```diff
@@ -14,7 +14,7 @@ def call(Object ctx, String imageStream, tag, name, value) {
                tags[i].annotations = [:]
            }
            tags[i].annotations[name] = value
-           echo "Annotating ImageStreamTag ${is.name()}:${tags[i].name()} with ${name}=${value}"
+           ctx.echo "Annotating ImageStreamTag ${is.name()}:${tags[i].name} with ${name}=${value}"
```

Comment:
It is not clear to me when you want echo and when you want this

Reply:
as a matter of good hygiene, if outside of the Jenkinsfile, I use the Jenkinsfile context

Reply:
that wasn't the issue though... the problem is that it was throwing an error because tags[i].name is not a function.